pyhdb/cesu8.py

```diff
@@ -152,7 +152,7 @@ class StreamReader(codecs.StreamReader):
 
 
 def search_function(encoding):
-    if encoding == 'cesu-8':
+    if encoding == 'cesu8':
```

Comment:
```suggestion
    if encoding in ('cesu8', 'cesu_8', 'cesu-8'):
```

Reply:
This is the only needed change. No need to replace all "cesu-8" with "cesu8".

Reply:
> This is the only needed change. No need to replace all "cesu-8" with "cesu8".

(Though I didn't try your suggestion,) You're right! Thank you for comment!